support/vim/install.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Instead of chaining things up with &&, start the script off with 'set -eu'.  e will stop the script on any error. u will error on undefined variables (not helpful here, but this is general advice for all scripts.)

then everything else can just be ordinary lines